client/verta/verta/tracking/_role.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+from .._protos.public.uac import RoleV2_pb2 as _Role
+
+
+class Role(object):
+    """
+    Object representing a Role.
+
+    """
+
+    def __init__(self, conn, msg):
+        self.conn = conn
+        self.msg = msg
+        self.id = msg.id
+        self.org_id = msg.org_id
+        self.name = "{}:{}".format(msg.org_id, msg.name)
+
+    @classmethod
+    def _create(
```

Comment:
There seems to be some inconsistency in the naming conventions we use in these cases.  Elsewhere, we would call this `_create_proto` since it contains that actual proto-request operation.  Perhaps that only exists where it was necessary to nest the request operation inside some additional logic, but it would be good to stay consistent, whatever we choose.

Reply:
renamed